index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   
+    <header id="nav-bar">
+        <a href="#"><h1>
+              Sonu
+            </h1></a>
+            <desktop id="header-desktop">
+            <h2 id="portfolio">Portoflio</h2>
+            <h3 id="about-header">About</h3>
+            <h4 id="contact-header">Contact</h4>
+            <img src="images/letter-icon.svg" id="letter-icon">
+           <a href="#"><img src="images/Union.png" id="menu"></a>
+        </desktop>
+    </header>
+    <section id="home">
+        
+        <div id="intro">
+        <h1>
+            Hey There. I'm Sonu
+        </h1>
+        <h2>
+            I’am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+        </p>
+      </section>
```

Comment:
Kindly add the social media icons to your page just as they are in the figma design. This is the figma design with the icons:
<img width="374" alt="Screenshot 2022-10-29 at 16 06 59" src="https://user-images.githubusercontent.com/48546992/198834039-19668840-0e5d-4e90-af34-80ef3052ed34.png">

Your page does not have these
<img width="414" alt="Screenshot 2022-10-29 at 16 12 44" src="https://user-images.githubusercontent.com/48546992/198834001-3136d24f-ead1-4ab2-bdc9-aa4997ee9cbb.png">